feat: update readme (#1)
eve-autonomy/engage_srv_converter: README.md

> # engage_srv_converter

Reviewer: ```suggestion
# engage service converter
```

## Overview

> This node acts as an interface to the autoware and outputs the input of pressing the departure button as the departure instruction of the autoware.

Reviewer: ```suggestion
This node converts a button action from ros topic to service for the Autoware interface.
```

## Input and Output
- input

>   - from engage_/[button_manager](https://github.com/eve-autonomy/button_manager)

Reviewer: ```suggestion
  - from engage_/[button_manager](https://github.com/eve-autonomy/button_manager/)
```

>     - `engage_button_manager/output/engage_button` : Departure button input. (this topic is remapping from /input/engage_button)

Reviewer: ```suggestion
    - `engage_button_manager/output/engage_button` : Departure button input. (this topic is remapped from `/input/engage_button`.)
```

- output

>   - to [autoware_state_machine](https://github.com/eve-autonomy/autoware_state_machine)

Reviewer: ```suggestion
  - to [autoware_state_machine](https://github.com/eve-autonomy/autoware_state_machine/)
```

>     - `/api/external/set/engage` : Departure instructions to Autoware. It is provided by the ROS service, not the ROS msg.

Reviewer: ```suggestion
    - `/api/external/set/engage` : Departure instructions to Autoware. This interface type is ROS2 service, not ROS2 message.
```
